### PR TITLE
May 2017 update

### PR DESCRIPTION
**Pull request by seanwestfall**
Update to the _data.json file. Still need to update the sponsors info.

### CODE REVIEW

File: public/_data.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-      "datetime": "Thursday April 27th, 2017 7pm",
-      "date": "4/27/2017 19:00:00",
+      "datetime": "Thursday May 25th, 2017 7pm",
+      "date": "5/25/2017 19:00:00",
       "host": {
-        "address": "2415 Michigan Ave, Santa Monica, CA",
-        "name": "Headspace",
-        "misc": "Street parking nearby, and next to the 26th St/Bergamot station 🚃",
-        "url": "https://www.headspace.com/join-us"
+        "address": "2301 Rosecrans Ave, El Segundo, CA 90245",
```
Comment (jcblw) [inline public/_data.json]: This is not the correct location,

`3211 Olympic Blvd, Santa Monica, CA 90404` is

**Comment by ashwell** [comment]
looks like the headspace sponsor spot needs to be replaced with SapientRazorfish_

**Comment by jcblw** [comment]
nice catch @ashwell I have not heard back from any sponsors so we should probably announce with one. @seanwestfall do you have access to ti.to? We need to create the next event if not I can create it.

**Comment by seanwestfall** [comment]
@jcblw You have not showed me how to update ti.to. Yes, you will have to show me how to update and access ti.to.

**Comment by seanwestfall** [comment]
@ashwell @jcblw Also, I know that I need to update the sponsor slot with Razorfish, that's comping up shortly, just need to resize the logo a bit. I'll bring it up on slack in the morning.

File: public/_data.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-      "date": "4/27/2017 19:00:00",
+      "datetime": "Thursday May 25th, 2017 7pm",
+      "date": "5/25/2017 19:00:00",
       "host": {
-        "address": "2415 Michigan Ave, Santa Monica, CA",
-        "name": "Headspace",
-        "misc": "Street parking nearby, and next to the 26th St/Bergamot station 🚃",
-        "url": "https://www.headspace.com/join-us"
+        "address": "3211 Olympic Blvd, Santa Monica, CA 90404",
+        "name": "Sapient Razor Nitro Fish Lizard",
```
Comment (ashwell) [inline public/_data.json]: @jcblw let's not confuse @seanwestfall and make sure we have the right name here...

Comment (jcblw) [inline public/_data.json]: 👀

**Comment by seanwestfall** [comment]
@ashwell requested fixes made.

**Comment by jcblw** [comment]
awesome @seanwestfall and @ashwell  gonna make the tito tonight and insert in this pr.

**Pull request merged**